### PR TITLE
Add GraphiQL route to app to let users experiment with test queries

### PR DESCRIPTION
**Pull request by ernieturner**
Adds the GraphiQL playground UI to the `/playground` route when running the `newsfeed` app locally. Nothing fancy so far with plugins or anything, but works well in terms of pulling schema docs.

Screenshot 2023-01-13 at 10 56 37 AM

### CODE REVIEW

**Comment by ernieturner** [comment]
@davidmccabe Just a heads up about this addition

**Review by sibelius — APPROVED** [review]

File: newsfeed/src/components/playground/GraphiQL.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import "regenerator-runtime/runtime.js";
```
Comment (alunyov) [inline newsfeed/src/components/playground/GraphiQL.tsx:1]: hmm, why do we need this?

**Review by alunyov — CHANGES_REQUESTED** [review]
This is great!
Thanks @ernieturner for adding this so quickly!

Overall, looks good - one small question about the `regenerator-runtime/runtime.js`. I think we are using webpack for the example, and it should be able to compile graphiql without any additional runtimes. But I'm missing something...

File: newsfeed/src/components/playground/GraphiQL.tsx (hunk 1 above)
Comment (ernieturner) [reply newsfeed/src/components/playground/GraphiQL.tsx]: Good catch, removed. I think this just got added when I was copying in some GraphiQL examples.

**Review by alunyov — APPROVED** [review]

**Review by vit0rr — APPROVED** [review]

**Pull request merged**